package.json

```diff
@@ -111,7 +111,7 @@
 		]
 	},
 	"engines": {
-		"node": ">=4.0"
+		"node": ">=8.0"
```

Comment:
Can't do that. Breaking change.

Reply:
I thought we were going to min version of `10.2.0` now?

Reply:
We are, but **this** is a breaking change. You need to bump the major and we're not bumping the major so you can use default parameters.

Reply:
When are we bumping version for v5.  I thought that was coming soon as it was required for SDK 9. https://jira.appcelerator.org/browse/TIMOB-27233

Reply:
Titanium CLI v6 will not be based off Titanium CLI v5 code.